conf.py

```diff
@@ -13,13 +13,21 @@
 # Sphinx extension module names and templates location
 sys.path.append(os.path.abspath("_extensions"))
 extensions = [
-    "gdscript",
-    "godot_descriptions",
-    "sphinx_search.extension",
     "sphinx_tabs.tabs",
     "sphinx.ext.imgmath",
 ]
 
+sphinx_tabs_nowarn = True
```

Comment:
What does this do?

Reply:
The code tabs extension produces a warning that it's not compatible with the current builder if that builder is not in a list of "compatible" builders (like the HTML builder is). It then does nothing either way.

It's not a useful warning, as either you use a builder like the HTML builder and it works, or you use our nothing-builder and it does... nothing. :) I'll add a comment.

Reply:
Done.